KubeArmor/feeder/policyMatcher.go

```diff
@@ -921,6 +921,50 @@ func (fd *Feeder) UpdateDefaultPosture(action string, namespace string, defaultP
 	}
 }
 
+// MatchResources function
+func matchResources(secPolicy tp.MatchPolicy, log tp.Log) bool {
+	// match process and file resources
+
+	firstLogResource := strings.Split(log.Resource, " ")[0]
+	firstLogResourceDir := getDirectoryPart(firstLogResource)
+	firstLogResourceDirCount := strings.Count(firstLogResourceDir, "/")
+
+	// secondLogResourceDir to handle cases where we don't have `/` at the end of path for eg:- "/run/secrets"
```

Comment:
Block /run/secrets/
MatchPrefix and log.Resource +"/" == secPolicy.Dir

Please rethink the refactor